### ISSUE TITLE
Running application from scratch fails

### ISSUE THREAD

**Issue by Glamdring** [openregister/openregister-java#342]
Hello,

I tried running the openregister-java application locally, using `sudo ENVIRONMENT=beta REGISTERS=country ./run-application.sh` but it failed to start.

Initially it reports:

```
Waiting for HTTP on :8081
Loading register...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  4491    0  4491    0     0  12452      0 --:--:-- --:--:-- --:--:-- 12475
curl: (22) The requested URL returned error: 400 Bad Request
Stopping and removing containers...
```
Then I added --verbose to the curl command that fetches the fsf in run-application.sh:

`--data-binary @<(curl --verbose "https://$register.$DOMAIN/download-rsf")`

And that reported:

```
* Connection #0 to host field.register.gov.uk left intact
{"success":false,"message":"Exception when executing command: RegisterCommand{commandName='append-entry', arguments=[system, register:field, 2016-08-04T14:45:41Z, sha-256:ebb8bc0c3639577c5b7d01e5b5a382c52dfc7dee41c290bd8b29647db00259d4]}","details":"Definition of register field does not match Register Register"}S
```

How can this be addressed and fixed?

As a sidenote: maybe explicitly point out in the readme that when cloning the repo, one should also init the submodules, otherwise the gradle build fails (as it doesn't find the govuk templates)

**Comment by fjavierm** [comment]
Hello,

The issue has been fixed. You can take a look to the PR #346.

Basically, the proper values for the beta environment have been set. And some intructions have been added to the readme to run it in environments other than beta.

**Comment by gidsg** [comment]
Hi @Glamdring 
Thanks for reporting this issue. We pushed a fix for this in #346 please try again after pulling master.
